app/src/components/market/sections/market_create/steps/items/funding_and_fee_step.tsx

```diff
@@ -324,22 +315,17 @@ const FundingAndFeeStep: React.FC<Props> = (props: Props) => {
                         <OutcomeItemText>{outcome.name}</OutcomeItemText>
                       </OutcomeItemTextWrapper>
                     </OutcomesTD>
-                    <OutcomesTD textAlign="right">{outcome.probability.toFixed(2)}%</OutcomesTD>
-                    <OutcomesTD textAlign="right">
-                      <TDFlexDiv textAlign="right">
-                        {formatBigNumber(sharesAfterInitialFunding[index], collateral.decimals)}
-                      </TDFlexDiv>
-                    </OutcomesTD>
+                    <OutcomesTD>{outcome.probability.toFixed(0)}%</OutcomesTD>
```

Comment:
This is a UI suggestion, so could use @pimato opinion, but I think we should include a decimal or two, at least if it's not a whole number.

Reply:
@pimato , what do you think about this?

Reply:
@KadenZipfel , fixed